docs/resources/organization.md

```diff
@@ -67,6 +67,9 @@ already exist in Grafana.
 - `editors` (Set of String) A list of email addresses corresponding to users who should be given editor
 access to the organization. Note: users specified here must already exist in
 Grafana unless 'create_users' is set to true.
+- `nones` (Set of String) A list of email addresses corresponding to users who who should be given none access to the organization.
```

Comment:
I would call it: `users_without_access` maybe. The others are self descriptive, this one, not so much. A 1:1 match with the API is not required